apcd-cms/src/apps/view_users/templates/view_user_edit_modal.html

```diff
@@ -0,0 +1,118 @@
+{% load static %}
+<link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
+<link rel="stylesheet" href="{% static 'view_users_table/css/modal.css' %}">
+<div id="editUserModal_{{r.user_id}}" class="modal fade" role="dialog">
+  <div class="modal-dialog modal-lg">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">Edit User ID: {{r.user_id}} for {{r.user_name}}</h4>
```

Comment:
```suggestion
        <h4 class="modal-title">Edit User: {{r.user_name}} ({{r.user_id}})</h4>
```
I had an idea for the header of the view & edit modals, let me know what you think!

Reply:
yes! Sounds great.